hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestHoodieBackedMetadata.java

```diff
@@ -925,6 +925,37 @@ public void testMetadataTableCompactionWithPendingInstants() throws Exception {
     assertEquals(HoodieInstantTimeGenerator.instantTimeMinusMillis(inflightInstant2, 1L), tableMetadata.getLatestCompactionTime().get());
   }
 
+  @Test
+  public void testInitializeMetadataTableWithPendingInstant() throws Exception {
+    init(COPY_ON_WRITE, false);
```

Comment:
If we are going to for only one table type, shall we test for MOR instead of COW? If the initialization works for MOR, it should work for COW as well.

Reply:
There should be no much difference for the table type, maybe we can just paramatize this test.